src/Resources/assets/shop/scss/pages/wishlist-button/index.scss

```diff
@@ -18,3 +21,17 @@
   }
 }
 
+.bb-add-to-cart-container{
+  display: flex;
+  gap: 1em;
+  padding: 1em 0;
+  
+  >*{
```

Comment:
missing spaces, this part should look something like that `> * {`

Reply:
And also was `*` selector the only option that worked here?

Reply:
That is selector that I needed to select every direct child of this element. I could use 'button' but it would break styles if child would have 'div' container etc.